README.md

```diff
@@ -18,7 +18,9 @@
         * Confirm installation of Docker Compose: `$ docker-compose --version`
 2. Download or [clone](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository-from-github/cloning-a-repository) repository.
 3. Run `nvm use` to ensure you are using the proper node version
-4. Install required dependencies by running `npm install` from within `/server` directory and from the `/client` directory.
+4. Install required dependencies:
+  -  run `npm install` from within `/server` directory and from the `/client` directory.
+  - run `npm run prepare` from within `/server` directory and from the `/client` directory.
```

Comment:
Since you are already here, can you change it to `npm ci` instead of `npm install`